tensorflow-opgen/src/TensorFlow/OpGen/ParsedOp.hs

```diff
@@ -67,6 +69,10 @@ data ParsedOp = ParsedOp
         -- Attributes which are list sizes (ints) that are inferred automatically
         -- from one or more of the input tensors.
         -- Associated with the list of tensors whose size it describes.
+    , parsedOpIsMonadic :: Bool
+        -- ^ Whether this op is stateful or takes a variable as input.  Such ops
```

Comment:
Is "variable" accurate here or any stateful input would do? At least the code (which reads really close to your comment!) suggests the latter.